README.md

```diff
@@ -2,12 +2,113 @@
 
 [![Build Status](https://travis-ci.org/ng-bootstrap/core.svg?branch=master)](https://travis-ci.org/ng-bootstrap/core)
 
-This library is a work in progress. Please check out our list of [issues](https://github.com/ng-bootstrap/core/issues) to see all the things we are implementing. Feel free to make any comment there ;)
+Welcome to the Angular 2 version of the []Angular UI Bootstrap](https://github.com/angular-ui/bootstrap) library.
+This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework.
 
-As for demos, we have a list of Plunkers with our components:
+As with Bootstrap 4 and Angular 2, this library is a work in progress. Please check out our list of [issues](https://github.com/ng-bootstrap/core/issues)
+to see all the things we are implementing. Feel free to make comments there.
+
+
+This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
+
+### Quick links
+- [Demo](#demo)
+- [Installation](#installation)
+    - [NPM](#install-with-npm)
+    - [Bower](#install-with-bower)
+    - [NuGet](#install-with-nuget)
+    - [Custom](#custom-build)
+    - [Manual](#manual-download)
+- [Support](#support)
+    - [FAQ](#faq)
+    - [Code of Conduct](#code-of-conduct)
+    - [PREFIX MIGRATION GUIDE](#prefix-migration-guide)
+    - [Supported browsers](#supported-browsers)
+    - [Need help?](#need-help)
+    - [Found a bug?](#found-a-bug)
+- [Contributing to the project](#contributing-to-the-project)
+- [Development, meeting minutes, roadmap and more.](#development-meeting-minutes-roadmap-and-more)
+
+# Demo
 
 * [Accordion](http://plnkr.co/edit/XuH7mXBycODO7KYgNSbH)
 * [Dropdown](http://plnkr.co/edit/8FNKrEURwSzqqtsK5Rxn?p=preview)
 * [Rating](http://plnkr.co/edit/2wtcZ5bmQk8xmvwWvFP6?p=preview)
 
-This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
+# Installation
+
+TBD
+
+#### Install with NPM
+
+TBD
+
+#### Install with Bower
+
+TBD
+
+#### Install with NuGet
+
+TBD
+
+#### Custom build
+
+TBD
+
+#### Manual download
```

Comment:
Do we want this section too? The only reason I can think of to support this is for Plunkers and such, but we should strongly adopt npm as the main mechanism for consuming the library.